tests/baselines/reference/api/typescript.d.ts

```diff
@@ -8750,7 +8750,6 @@ declare namespace ts {
     function isJSDocCommentContainingNode(node: Node): boolean;
     function isSetAccessor(node: Node): node is SetAccessorDeclaration;
     function isGetAccessor(node: Node): node is GetAccessorDeclaration;
-    /** True if has initializer node attached to it. */
```

Comment:
this one was copied over from an internal function declared above it, I don't think this comment is true as it made much more sense for the other function